tests/modules/sightings/test_models.py

```diff
@@ -108,95 +105,3 @@ def test_sighting_ensure_no_duplicate_encounters(db):
 
     test_sighting.delete()
     test_encounter.delete()
-
-
-@pytest.mark.skipif(extension_unavailable('edm'), reason='EDM extension disabled')
-def test_sighting_jobs(db, request, researcher_1):
```

Comment:
This test is covered by a different test that actually runs the jobs and checks their results.  There is no need to check the simulated accuracy of only the job schema